app/components/govuk_component/summary_list_card_component.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= tag.div(**html_attributes) do %>
+  <div class="govuk-summary-list__card-title-wrapper">
+    <%= tag.h2(title, class: "govuk-summary-list__card-title") %>
+
+    <% if actions.any? %>
+      <div class="govuk-summary-list__card-actions">
+        <ul class="govuk-summary-list__card-actions-list">
+          <% actions.each do |action| %>
+            <li class="govuk-summary-list__card-actions-list-item">
+              <%= action %>
+            </li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
+  </div>
+  <div class="govuk-summary-list__card-content">
+    <%= summary_list %>
```

Comment:
Should the `summary_list` be mandatory here? Can we display anything else in a card too, a paragraph for example ([like with Bootstrap's cards](https://getbootstrap.com/docs/5.2/components/card/)).